packages/dapp/src/components/Dashboard/DashboardActivity.tsx

```diff
@@ -327,10 +338,24 @@ class DashboardActivity extends React.Component<
           </Tab>
           <Tab title={<SubTabReclaimTokensText />}>
             <>
-              {/* TODO(jon): the value of `showTransferTokensMsg` should be populated from the TokenController */}
+              {/* TODO(sarah): the value of `showTransferTokensMsg` and `tutorialComplete` should be populated from the TokenController */}
               {this.state.showTransferTokensMsg && this.renderTransferTokensMsg()}
-              <ReclaimTokens onMobileTransactionClick={this.showNoMobileTransactionsModal} />
-              <DepositTokens />
+
+              {this.state.tutorialComplete ? (
+                <DashboardTransferTokenForm
+                  renderTransferBalance={this.renderTransferBalance}
+                  cvlAvailableBalance={balance}
+                  cvlVotingBalance={votingBalance}
+                >
+                  {this.state.transferTokenToVoting ? (
+                    <ReclaimTokens onMobileTransactionClick={this.showNoMobileTransactionsModal} />
```

Comment:
These are switched. Reclaiming tokens is transferring FROM the PLCR and depositing is transferring TO the voting contract.